finance/community-updates/2022-10.txt

```diff
@@ -0,0 +1,94 @@
+project team through the process of getting started and claiming funds.
+
+We're excited to be able to fund so many amazing projects! Thanks to those
+who submitted proposals and to everybody who took part in the discussion and
+voting process, and many congratulations to the successful project teams!
+
+The Finance Committee welcomes feedback on the funding process. Please feel
+free to reach out to us — see the contact details below — with your thoughts
+and suggestions for how it might be improved in the future.
+
+[1] https://github.com/astropy/astropy-project/pulls?q=label%3A%22funding-approved+💵%22+label%3A%22cycle+3%EF%B8%8F⃣%22+type%3Apr
+
+
+# Keeping Track of Work in Progress
+
+The Finance Committee uses GitHub issues to keep track of all the various ways
+in which we are allocating funds. If you would like to know where the
+Project's funds are being spent, check out the “finance” label in the
+astropy-project issue tracker [2].
+
+For previous funding rounds, we've used a single issue to keep track of all
+the various projects [3]. During Cycle III, we plan to use a single issue per
+project; watch out for these appearing over the next few days.
+
+[2] https://github.com/astropy/astropy-project/labels/finance%20%3Amoneybag%3A
+[3] https://github.com/astropy/astropy-project/issues/199/
+
+
+# Open Collective
+
+Over the spring and summer of this year, NumFOCUS — the organization that
+manages our accounts — switched to using Open Collective [4] for handling
+budgeting and invoicing. This new system is a substantial step forward — it's
+easier to use and more transparent than its predecessor.
+
+All invoices must now be submitted through Open Collective. This process is
+straightforward, but it is important that you know which of our various
+funding lines (or “projects” in Open Collective's jargon) you should submit
+your expense to (NASA, Moore, etc). If you aren't sure, reach out to the
+Finance Committee (details below) to check. When you know which project to
+use, head over to the Astropy area on Open Collective [5], select the right
+project, and then hit “submit expense”.
+
+As always, don't hesitate to reach out if you have questions.
+
+[4] https://opencollective.com
+[5] https://opencollective.com/astropy
+
+
+# Applying for Funding
+
+Although the Cycle III proposal process has now been completed, it's still
+possible to apply to the Finance Committee for funding. We welcome requests
+following the guidelines at [6].
+
+[6] https://github.com/astropy/astropy-project/blob/main/finance/process/request-funding.md
+
+
+# Finance Committee Contacts
+
+You can learn more about the Finance Committee's business and how you can get
+in touch in the astropy-project repository [7]. In particular, you can keep
+track of what we're working on by following our running notes [8], or reach
+out to us on Slack (#finance-committee), by filing a GitHub issue, or by mail
```

Comment:
```suggestion
out to us on Slack (#finance-committee), by filing a GitHub issue, or by e-mail
```

Reply:
I thought I was the only person left in the entire world who hyphenates e-mail!